.github/workflows/deploy-garmin-sync.yml

```
@@ -132,6 +142,46 @@ jobs:
             --min-instances=0 --max-instances=1 \
             --allow-unauthenticated
 
+      - name: Deploy Google Health account linker
+        if: ${{ secrets.GOOGLE_HEALTH_CLIENT_ID != '' && secrets.GOOGLE_HEALTH_CLIENT_SECRET != '' }}
```

actionlint (1.7.12)

[error] context "secrets" is not allowed here. available contexts are "env", "github", "inputs", "job", "matrix", "needs", "runner", "steps", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)